components/director/pkg/scope/directive_test.go

```diff
@@ -40,7 +40,9 @@ func TestHasScope(t *testing.T) {
 		// WHEN
 		_, err := sut.VerifyScopes(ctx, nil, nil, fixScopesDefinition())
 		// THEN
-		assert.EqualError(t, err, "insufficient scopes provided [required=read;write; actual=delete]")
```

Comment:
Judging by this fix, I guess in error messages returned to user it is non-deterministic whether `required` or `actual` will be first? Correct me if I'm wrong.